easy_my_coop/demo/coop.xml

```diff
@@ -70,7 +70,7 @@
     </record>
 
     <record id="account_cooperator_demo" model="account.account">
-        <field name="code">416101</field>
+        <field name="code">416000</field>
         <field name="name">Cooperators</field>
         <field name="user_type_id" ref="account.data_account_type_receivable"/>
         <field name="reconcile" eval="True"/>
```

Comment:
maybe that this should go in easy_my_coop_l10n_be...

Reply:
But I need it to initiate a demo db, I don't want to depend on l10n_be. What could be a good value for demo data?